modules/doc/content/getting_started/installation/inl_hpc_install_moose.md

```diff
@@ -0,0 +1,54 @@
+# INL HPC Cluster
+
+This page aims at simplifying the installation process on INL computing clusters.
+The [general cluster instructions](hpc_install_moose.md) may also be used, but these
+instructions will save some steps.
+
+## Module-based installation
+
+### Step 1: load some compilers
+
+In order to compile MOOSE and its applications, we first need to load compilers & compiling tools.
+The following modules are recommended for each of INL's clusters:
+
+- Falcon: `module load MVAPICH2/2.2-GCC-5.4.0-2.28 CMake`
+
+- Lemhi: `module load mvapich2/2.3.6-gcc-11.2.0-w2fi cmake`
+
+- Sawtooth: `module load mvapich2/2.3.5-gcc-8.4.0 cmake`
+
+
+### Step 2: module-load or download MOOSE dependencies
+
+MOOSE dependencies, mainly Petsc and libmesh, may be obtained using the `update_and_rebuild_...`
+scripts, as is detailed in these [instructions](hpc_install_moose.md).
+
+They may also be obtained directly from HPC modules pre-installed on each machine:
+
+- Falcon: Unavailable
+
+- Lemhi: `module load PETSc/3.15.1-gcc-11.2.0`
+
+- Sawtooth: Currently unavailable
+
+
+TODO: add libmesh, double check the petscs
+
+
+### Step 3: module-load or download MOOSE and/or the application you need
+
+MOOSE may be classically obtained from its [GitHub repository](https://github.com/idaholab/moose).
+Many open-source MOOSE applications may also be obtained from GitHub, while others will require
+officially requesting access through [the NCRC](ncrc/ncrc_ondemand.md). Please then follow the
+associated instructions for installation.
+
+However, if you do not plan to develop, you may load pre-built binaries for MOOSE
+and the applications you have access to by running: `module load use.moose moose-apps <app_name>`.
+Pre-built executables can then be found in: XXXXXXXXXX
+
+## Not recommended: conda installation
+
+The [conda-based installation](conda.md) is not recommended for INL HPC, as it does not use
+optimized, architecture-specific compilers. However, if you are not seeking performance,
+and will not use more than one node (to limit the attrition of computational resources),
+you are free to use them.
```

Comment:
👍

Reply:
I like this (agreeing with @milljm), but I would suggest making this a yellow warning tip box, so that it stands out.

Reply:
ok just updated. 
Will be a lot of boxes on this page but not really a problem

Reply:
Yeah, I think that's fine if it actually helps clarify